2023-01-31 21:21:22.222764
--- Logging error ---
Traceback (most recent call last):
  File "C:\Program Files\Python39\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
  File "C:\Elsiros\controllers\Robofest_TEAM\main_pb.py", line 157, in write
    self.out.WriteText(string)
wx._core.wxAssertionError: C++ assertion "m_count == -1 || m_count == -2" failed at ..\..\src\msw\textctrl.cpp(304) in UpdatesCountFilter::UpdatesCountFilter(): wrong initial m_updatesCount value
Call stack:
  File "C:\Program Files\Python39\lib\threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
  File "C:\Program Files\Python39\lib\threading.py", line 973, in _bootstrap_inner
    self.run()
  File "C:\Program Files\Python39\lib\threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Elsiros\controllers\player\communication_manager_robokit.py", line 252, in run
    self.__update_history(message)
  File "C:\Elsiros\controllers\player\communication_manager_robokit.py", line 66, in __update_history
    self.logger.warning(f"WARNING! Large protobuf time rx delta = {delta}")
Message: 'WARNING! Large protobuf time rx delta = 2390'
Arguments: ()
2023-01-31 21:21:25,829 - [INFO] - __main__ - (class_Motion_Webots_PB.py).imu_activation(92) - imu_activation
